python/cuml/test/test_fil.py

```diff
@@ -572,11 +585,8 @@ def test_lightgbm(tmp_path, num_classes, n_categorical):
                                  n_estimators=num_round)
         lgm.fit(X_fit, y)
         lgm.booster_.save_model(model_path)
-        fm = ForestInference.load(model_path,
-                                  algo='TREE_REORG',
-                                  output_class=True,
-                                  model_type="lightgbm")
-        lgm_preds = lgm.predict(X)
+        lgm_preds = lgm.predict(X).astype(int)
+        fm = fil_load()
         assert array_equal(lgm.booster_.predict(X).argmax(axis=1), lgm_preds)
         assert array_equal(lgm_preds, fm.predict(X))
```

Comment:
Does `array_equal` work with different types (ints and floats)?

Reply:
yes, it correctly compares them